### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by ishayrao**

### CODE REVIEW

File: coco_models/src/models/faster_rcnn_detector.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class FasterRCNNDetector(nn.Module):
+    def __init__(self, cfg: DictConfig):
+        super(FasterRCNNDetector, self).__init__()
+        
+        # Load pretrained model withOUT default weights
+        weights = FasterRCNN_MobileNet_V3_Large_FPN_Weights.DEFAULT
+        self.model = fasterrcnn_mobilenet_v3_large_fpn(
+            weights=None,
+            box_nms_thresh=0.5,     # NMS IoU threshold
+            box_detections_per_img=10  # Max detections per image
```
Comment (Rob1in) [inline coco_models/src/models/faster_rcnn_detector.py:18]: does this value make sense wrt to your dataset?

**Comment by Rob1in** [comment]
Very good work. Thank you for sharing results and good visualizations tools. A few stuffs:

1. Add how to visualize losses in tensorboard in the README. 
2. A good question to ask yourself: what are the hyperparameters that should be validated with Optuna? For almost every hard coded value. The main ones that I am missing are weights for the 4 terms of the summed loss function. Normalization parameters too? 
3. Let's add intensity-based data augmentation, I am more interested in those rather than geometrical ones because it will make the model more robust to other IR camera.
4. Add the ability to train and evaluate on multiple dataset. Your 2 dataset classes are good, let's make a third one that returns "mixed" image from different dataset. 
5. Add specific instructions on how to get the dataset and where to put them so the code works. 


Let's merge this for now and plan work to implement those. Good job again, let's see how to raise those metrics a bit more.

**Review by Rob1in — APPROVED** [review]
